src/client.rs

```diff
@@ -191,6 +192,29 @@ pub trait Histogrammed {
         key: &'a str,
         value: u64,
     ) -> MetricBuilder<'_, '_, Histogram>;
+
+    /// Record a single histogram value with the given key.
+    ///
+    /// The duration will be converted to nanoseconds. If the duration cannot
+    /// be represented as a `u64` an error will be returned. Although
+    /// histograms don't necessarily represent times, this method is provided
+    /// as a convenience.
```

Comment:
It may be worth pointing out that Datadog does consider histograms to be a superset of times.

> Datadog’s `HISTOGRAM` metric type is an extension of the StatsD timing metric type

[source](https://docs.datadoghq.com/developers/metrics/types/?tab=histogram#metric-type-definition)

Reply:
Good catch, thanks!